src/Components/Search/Suggestions/SuggestionItem.tsx

```diff
@@ -121,19 +121,37 @@ const QuickNavigation: React.FC<{
 }> = ({ href, showArtworksButton, showAuctionResultsButton }) => {
   const { trackEvent } = useTracking()
 
-  const handleQuickNavigationClick = (type: "artworks" | "auction-results") => {
-    const label = type === "artworks" ? "Artworks" : "Auction Results"
-    const destinationPath =
-      type === "artworks" ? "/works-for-sale" : "/auction-results"
+  const handleArtworksItemClicked = () => {
+    trackEvent(
+      tracks.quickNavigationItemClicked({
+        destinationPath: "works-for-sale",
+        label: "Artworks",
+      })
+    )
+  }
+
+  const handleAuctionResultsItemClicked = () => {
+    trackEvent(
+      tracks.quickNavigationItemClicked({
+        destinationPath: "auction-results",
+        label: "Auction Results",
+      })
+    )
+  }
 
-    const trackingEvent: SearchSuggestionQuickNavigationItemSelected = {
+  const tracks = {
```

Comment:
can we take this constant out of the scope of the component?

Reply:
we could but then we're gonna have to pass href as input there as well

Reply:
I think it's better to do this. We can pass full url in `destinationPath`

```javascript
trackEvent(
  tracks.quickNavigationItemClicked({
    destinationPath: `${href}/${auction-results}`,
    label: "Auction Results",
  })
)

const tracks = {
  quickNavigationItemClicked: ({
    href,
    label,
  }: {
    destinationPath: string
    label: "Auction Results" | "Artworks"
  }): SelectedSearchSuggestionQuickNavigationItem => ({
    context_module: ContextModule.header,
    destination_path: destinationPath,
    action: ActionType.selectedSearchSuggestionQuickNavigationItem,
    label,
  }),
}
```

Reply:
unless we do something like this
```
  const handleArtworksItemClicked = () => {
    trackEvent(
      tracks.quickNavigationItemClicked({
        destinationPath: `${href}/works-for-sale`,
        label: "Artworks",
      })
    )
```

if it looks okay I'm going for it

Reply:
yes, it looks great. thanks